README.md

```diff
@@ -0,0 +1,51 @@
+Install postresql server
+```
+apt install postgresql
+```
+
+Create a postgres user and db as root:
+```
+createuser <your username>
+createdb -O <your username> tunnistamo
+```
+
+Clone the repo
+
+```
+git clone git@github.com:City-of-Helsinki/tunnistamo.git
+cd tunnistamo
+```
+
+Initiate a virtualenv
+```
+pyenv virtualenv tunnistamo-env
+pyenv local tunnistamo-env
+```
+
+Install the Python requirements
+```
+pip install prequ
+rm requirements.txt
+prequ update
+pip install -r requirements.txt (don't use `prequ sync`)
```

Comment:
When displaying shell code, do comments like this:
```bash
pip install ... # don't use `prequ sync`
```
You can mark the code blocks to have syntax highlighting by starting the block: ```bash